src/components/Stack/Stack.tsx

```diff
@@ -1,20 +1,32 @@
 import React from 'react';
 import classNames from 'classnames';
 import './Stack.scss';
+import * as CSS from 'csstype';
+
+const defaultProperty = 'flex-start';
 
 export type StackProps = React.HTMLAttributes<HTMLDivElement> & {
   space: number;
   direction?: 'vertical' | 'horizontal';
+  alignItems?: CSS.AlignItemsProperty;
+  justifyContent?: CSS.JustifyContentProperty;
+  style?: undefined;
 };
 
 export const Stack: React.FC<StackProps> = ({
   className,
   children,
   space,
   direction = 'vertical',
+  alignItems = defaultProperty,
+  justifyContent = defaultProperty,
```

Comment:
Do you need to default these? I think we can leave them uninitialized when not passed.